ConfigSpace/configuration_space.py

```diff
@@ -90,23 +102,38 @@ def __init__(
         self._children_of = dict()
 
     def generate_all_continuous_from_bounds(self, bounds: List[List[Any]]) -> None:
+        """
+        Generates Unniform Float Hyperparameter from a list containing tuples 
+        with lower and upper bounds. The generated hyperparameters are added to 
+        the configuration space.
+        
+        Parameters
+        ----------
+        bounds : list(float,float)
+            List containing tuples with lower and upper bounds
+        Returns
+        -------
+            None
+        """
         for i, (l, u) in enumerate(bounds):
             hp = ConfigSpace.UniformFloatHyperparameter('x%d' % i, l, u)
             self.add_hyperparameter(hp)
 
     def add_hyperparameters(self, hyperparameters: List[Hyperparameter]) -> List[Hyperparameter]:
-        """Add hyperparameters to the configuration space.
+        """
+        Add hyperparameters to the configuration space.
 
         Parameters
         ----------
-        hyperparameters : list
-            List of hyperparameters to add.
+        hyperparameters : list[:ref:`Hyperparameters`]
```

Comment:
Shouldn't it be :ref:`Hyperparameter`?

Reply:
Unfortunately, it was not really possible to make a link to the base class 'AbstractHyperparameter', because it is not mentioned in the Documentation. 

Therefore I ' ve created a link in the API doc: 
 ```
.. _Hyperparameters:

Hyperparameters
===============
```
... that references the hyperparameter from the docstrings to the doc section with all hyperparameters.
Is this something, that is okay for you?